packages/workbox-precaching/utils/createCacheKey.mjs

```diff
@@ -0,0 +1,64 @@
+/*
+  Copyright 2018 Google LLC
+
+  Use of this source code is governed by an MIT-style
+  license that can be found in the LICENSE file or at
+  https://opensource.org/licenses/MIT.
+*/
+
+import {WorkboxError} from 'workbox-core/_private/WorkboxError.mjs';
+
+import '../_version.mjs';
+
+// Name of the search parameter used to store revision info.
+const REVISION_SEARCH_PARAM = '_wbRevision';
```

Comment:
Do you think there's any value in making this customizable? My initial thought is *no*, since it would make upgrading harder, but what do you think?

Also, I wonder if it makes sense to make it something that's even less likely to collide and be more obvious what it is when looking at devtools, e.g. `__WB_REVISION__`.

Reply:
I don't see there being a risk of query parameter collision, so I'd rather not make it customizable.

I can change to `'__WB_REVISION__'`.